test/special/eslint.js

```diff
@@ -97,21 +97,30 @@ const testCases = [
     expected: [],
   },
   {
-    name: 'handle config of scoped module',
```

Comment:
According to the doc, we still need to support this test case. Could you please add it back? (And polish the name is a plus. 😄 )